template/report.html.tmpl

```diff
@@ -1,441 +1,544 @@
-    {{ with .RelayMetrics -}}
-    <h3>{{ .sdk.name }} {{ .sdk.version }}</h3>
-    {{- end }}
-    <!--Div that will hold the chart-->
-    <div id="chart_div">Loading...</div>
-    <p>
-      Percentile range:
-      <input
-        type="range"
-        class="slider-width500"
-        min="1"
-        max="8"
-        value="7"
-        step="1"
-        width="300px"
-        onchange="showValue(this.value)"
-      />
-      <span id="percentileRange">99.99999%</span>
-    </p>
-
-    <!-- Information about the run -->
-    {{ range .Data }}
-    <details>
-      <summary>{{ .Name }} Run Details</summary>
-      {{ with .TestResult.Metrics -}}
-      <div class="runDetails">
-        <h4>Errors</h4>
-        <ul>
-        {{ range .Errors }}
-          <li>{{ . }}</li>
-        {{ else }}
-          <li>No Errors</li>
-        {{ end }}
-        </ul>
+      return series;
+    }
+
+    // Reacts to Slider Input: Updates Chart
+    function showValue(newValue) {
+      const newPercentile = Math.pow(10, newValue);
+      const percentile = 100.0 - 100.0 / newPercentile;
+      document.getElementById("percentileRange").innerHTML = percentile + "%";
+      maxPercentile = newPercentile;
+      drawChart();
+      return { typed: "" };
+    }
+
+    function copy(className, runName) {
+      const hdrNodes = document.querySelectorAll(`.${className}`);
+      hdrNodes.forEach(node => {
+        if (node.getAttribute("data-name") === runName) {
+          navigator.clipboard.writeText(node.innerHTML);
+          const snack = document.getElementById("snackbar");
+          snackbar.innerHTML = `Copied ${className} for ${runName}`
+          // Add the "show" class to DIV
+          snackbar.className = "show";
+          setTimeout(() => {
+            snackbar.className = snackbar.className.replace("show", "");
+            snackbar.innerHTML = "";
+          }, 1000);
+        }
+      })
+    }
+  </script>
+</head>
+
+<body>
+  <header>
+    <h1>Sentry SDK Benchmark Report</h1>
+  </header>
+
+  <nav>
+    <ul>
+      <li><a href="#configuration">Configuration</a></li>
+      <li><a href="#latency">Latency</a></li>
+      <li><a href="#memory-cpu">Memory & CPU Usage</a></li>
+      <li><a href="#network">Network Traffic</a></li>
+      <li><a href="#debug">Debugging</a></li>
+    </ul>
+  </nav>
+
+  <main>
+    <section>
+      <h2 id="configuration">Configuration</h2>
+      <div>
+        <h3>Target Application</h3>
+        <dl>
+          <div>
+            <dt>Language</dt>
+            <dd>{{ .Title }}</dd>
+          </div>
+          <div>
+            <dt>Framework</dt>
+            <dd><a href="https://github.com/getsentry/sentry-sdk-benchmark/tree/main/platform">{{ "TODO" }}</a></dd>
+          </div>
+          {{ with .RelayMetrics -}}
+          <div>
+            <dt>Sentry SDK version</dt>
+            <dd><a href="https://github.com/getsentry/{{ .sdk.name }}/releases/{{ .sdk.version }}">{{ .sdk.version }}</a></dd>
```

Comment:
This will have to be processed. We can usually just replace `.` with `-` probably.

```go
"formatSDKName": func(n string) string {
	return strings.ReplaceAll(n, ".", "-")
}
```

and then

```suggestion
            <dd><a href="https://github.com/getsentry/{{ formatSDKName .sdk.name }}/releases/{{ .sdk.version }}">{{ .sdk.version }}</a></dd>
```

Reply:
Yeah I know this produces broken links for now. It has to change, but I just wanted to have the link there as a reminder of intention.

Possibly the version number will also not work for all SDKs because some have a `v` prefix.